x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/scalar/multivalue/MvMedian.java

```diff
@@ -106,7 +106,7 @@ static class Doubles {
         public int count;
     }
 
-    @MvEvaluator(extraName = "Double", finish = "finish")
+    @MvEvaluator(extraName = "Double", finish = "finish", ascending = "ascending")
```

Comment:
This evaluator was missing the "ascending" value, so it wasn't calling the optimized ascending function. Just a minor unrelated """fix"""